packages/contracts-bedrock/test/L1/OPContractsManagerStandardValidator.t.sol

```diff
@@ -1541,24 +1439,11 @@ contract OPContractsManagerStandardValidator_FaultDisputeGame_Test is OPContract
     /// @notice Tests that the validate function successfully returns the right error when the
     ///         FaultDisputeGame (permissionless) version is invalid.
     function test_validate_faultDisputeGameInvalidVersion_succeeds() public {
-        vm.mockCall(address(fdgImpl), abi.encodeCall(ISemver.version, ()), abi.encode("0.0.0"));
-        if (isDevFeatureEnabled(DevFeatures.CANNON_KONA)) {
-            assertEq("PLDG-20,CKDG-20", _validate(true));
-        } else {
-            assertEq("PLDG-20", _validate(true));
-        }
-    }
-
-    /// @notice Tests that the validate function successfully returns the right error when the
-    ///         FaultDisputeGame (permissionless) game type is invalid.
-    function test_validate_faultDisputeGameInvalidGameType_succeeds() public {
-        // For v2 game contracts, we don't store the game type anywhere other than the DGF gameImpls and gameArgs maps
-        // So, there's not really an obvious way to return an invalid gameType
-        skipIfDevFeatureEnabled(DevFeatures.DEPLOY_V2_DISPUTE_GAMES);
-        vm.mockCall(
-            address(fdgImpl), abi.encodeCall(IDisputeGame.gameType, ()), abi.encode(GameTypes.PERMISSIONED_CANNON)
-        );
-        assertEq("PLDG-30", _validate(true));
+        BadVersionReturner bad = new BadVersionReturner(standardValidator, ISemver(address(pdgImpl)), "0.0.0");
```

Comment:
Note that installing mocks via vm.mockCall on the dispute game implementations is ineffective. Because both std validator and opcm DG implementations have the same addresses. And neither is a proxy.